pool/src/lib.rs

```diff
@@ -267,6 +406,218 @@ impl LendingPool {
         //TODO: implement when rates exists
         //balance_decrease_allowed()
     }
+
+    fn validate_borrow(
+        env: &Env,
+        who: Address,
+        reserve: &ReserveData,
+        user_config: &UserConfiguration,
+        amount: i128,
+        amount_in_xlm: i128,
+    ) -> Result<(), Error> {
+        assert_with_error!(env, amount != 0, Error::InvalidAmount);
```

Comment:
Should we check if `amount` is negative?

Should we also check `amount_in_xlm`?